README.md

```diff
@@ -94,3 +117,7 @@ unit and functional tests. For example:
     mvn test -D test=com.stripe.model.AccountTest
     mvn test -D test=com.stripe.functional.ChargeTest
     mvn test -D test=com.stripe.functional.ChargeTest#testChargeCreate
+
+<!--
+# vim: set tw=79:
+-->
```

Comment:
I snuck another one of these in :$ IMO it's a good idea to try to get these in a state where they're easily readable as either plain text or rendered Markdown on GitHub, and consistent line length is a piece of that.